specs/RuntimeSpec/Receipts.md

```diff
@@ -8,9 +8,9 @@ Each receipt has a [`predecessor_id`](#predecessor_id) (who sent it) and [`recei
 Receipts are one of 2 types: action receipts or data receipts.
 
 Data Receipts are receipts that contains some data for some `ActionReceipt` with the same `receiver_id`.
-Data Receipts has 2 fields: the unique data identifier `data_id` and `data` the received result.
-`data` is an `Option` field and it indicates whether the result was a success or a failure. If it's `Some`, then it means
-the remote execution was successful and it contains the vector of bytes of the result.
+Data Receipts have 2 fields: the unique data identifier `data_id` and `data` the received result.
+`data` is an `Option` field and it indicates whether the result was a success or a failure. If it's `Some`, it means
+the remote execution was successful and it contains a result as a vector of bytes.
```

Comment:
"it contains a result as a vector of bytes" still feels a bit awkward. Do you think "it represents the result as a vector of bytes" would be better?

Reply:
that sounds great to me